### ISSUE TITLE
Output Format?

### ISSUE THREAD

**Issue by ngeldvis** [Britefury/django-labeller#12]
is the json output in a specific format?  I'm looking for COCO format

**Comment by licyj** [comment]
@ngeldvis  Yes, you can find out the specific output  format in "Image labeller notebook.ipynb"
